docs/helmrelease-guide/release-configuration.md

```diff
@@ -22,6 +22,31 @@ spec:
   helmVersion: v3
 ```
 
+## Migrating from Helm v2 to v3
+
+Helm Operator uses the [helm-2to3 plugin](https://github.com/helm/helm-2to3) under the hood to migrate HelmRelease objects.
+In order to perform a release conversion you have to:
+
+1. Set `.spec.helmVersion` to `v3`
+2. Add a migrate annotation `helm.fluxcd.io/migrate: "true"` (For testing, you can set the value to "dry-run" instead of "true")
+
+```yaml
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: redis
+  annotations:
+    helm.fluxcd.io/migrate: "true" # add annotation
+spec:
+  helmVersion: v3 # set helmVersion to v3
+  releaseName: redis
+  chart:
+    repository: https://kubernetes-charts.storage.googleapis.com
+    name: redis
+```
+
+After applying the new HelmRelease, the operator will take care of deleting the old v2 release that Tiller managed and converting it to the v3 format. Once you're satisfied with the migration, you can go ahead and remove the annotation. This approach allows teams to migrate their charts at scale to the v3 format without stopping the world.
```

Comment:
Can you please maintain the 80 character width?

Reply:
I get it. But the file was going up to 90 columns already. If you really want to enforce this, add it as a linting rule.